searchlib/src/vespa/searchlib/predicate/predicate_zero_constraint_posting_list.cpp

```diff
@@ -1,6 +1,7 @@
 // Copyright 2017 Yahoo Holdings. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
 
 #include "predicate_zero_constraint_posting_list.h"
+#include <vespa/vespalib/btree/btreeiterator.hpp>
```

Comment:
This should not be needed.  The relevant iterator should have been explicitly instantiated.